readthedocs/restapi/views/integrations.py

```diff
@@ -159,7 +167,13 @@ def handle_webhook(self):
         # Handle push events and trigger builds
         if event == GITHUB_PUSH:
             try:
-                branches = [self.data['ref'].replace('refs/heads/', '')]
+                branch = {
```

Comment:
I change branch to be a dict with `name` and `last_commit` as elements.
`last_commit` must have an `id` and `message`.